src/test/java/com/vinylteam/vinyl/web/servlets/SearchResultsServletTest.java

```diff
@@ -29,7 +29,7 @@ void doPostTest() throws IOException {
         when(mockedResponse.getWriter()).thenReturn(mockedPrintWriter);
 
         SearchResultsServlet searchResultsServlet = new SearchResultsServlet(mockedVinylService);
-        searchResultsServlet.doPost(mockedRequest, mockedResponse);
+        searchResultsServlet.doGet(mockedRequest, mockedResponse);
```

Comment:
@Test
    @DisplayName("Checks if all right methods are called")
    void doPostTest() throws IOException {

Reply:
Pls change name to doGet()